keyboards/massdrop/ctrl/keymaps/r-pufky/config.h

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
You can condense this whole file down to:

```c
/*
Copyright 2015 Jun Wako <wakojun@gmail.com>

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
*/

#pragma once

#define FORCE_NKRO

#define RGB_MATRIX_STARTUP_MODE RGB_MATRIX_SOLID_COLOR
#define RGB_MATRIX_STARTUP_VAL 100
#define RGB_BACKLIGHT_COLOR_1 RGB_BLUE
#define RGB_BACKLIGHT_COLOR_2 RGB_BLUE
```